### PR TITLE
LogRocket free tier

### PR DESCRIPTION
**Pull request by nickoferrall**
To remain within LogRocket's free tier, let's only record sessions where the user has encountered a fatal error

### CODE REVIEW

File: packages/client/components/ErrorBoundary.tsx [hunk 1]
```diff
@@ -25,15 +25,33 @@ class ErrorBoundary extends Component<Props, State> {
...
-      const email = (store as any)?._recordSource?._records?.[viewerId]?.email ?? ''
       Sentry.configureScope((scope) => {
         scope.setUser({email, id: viewerId})
       })
     }
-    LogRocket.captureException(error)
-    LogRocket.track('Fatal error')
+    const logRocketId = window.__ACTION__.logRocket
+    if (logRocketId) {
+      LogRocket.init(logRocketId, {
```
Comment (mattkrick) [inline packages/client/components/ErrorBoundary.tsx:37]: `window.localStorage.setItem(LocalStorageKey.ERROR_PRONE, true)`

File: packages/client/components/AnalyticsPage.tsx [hunk 2]
```diff
@@ -69,27 +68,6 @@ const AnalyticsPage = () => {
     }
   )
   const atmosphere = useAtmosphere()
-  useEffect(() => {
-    const logRocketId = window.__ACTION__.logRocket
-    if (logRocketId) {
```
Comment (mattkrick) [inline packages/client/components/AnalyticsPage.tsx:74]:
```
const errorProneAt = window.localStorage.getItem(LocalStorageKey.ERROR_PRONE_AT)
if (errorProneAt && new Date(errorProneAt) < new Date(Date.now() - ms('30d'))) {
  window.localStorage.deleteItem(LocalStorageKey.ERROR_PRONE_AT)
  return
} 

if (logRocketId && errorProne) ...
```

**Review by mattkrick — CHANGES_REQUESTED** [review]

File: packages/client/components/ErrorBoundary.tsx [hunk 3]
```diff
@@ -45,10 +46,11 @@ class ErrorBoundary extends Component<Props, State> {
         }
       })
       if (email) {
-        LogRocket.identify(atmosphere.viewerId, {
+        LogRocket.identify(viewerId, {
           email
         })
       }
+      window.localStorage.setItem(LocalStorageKey.ERROR_PRONE_AT, `${new Date().getTime()}`)
```
Comment (mattkrick) [inline packages/client/components/ErrorBoundary.tsx:53]: itty bitty the absolute most nitty: Date.now() is a little cleaner over new Date().getTime()

Comment (nickoferrall) [reply packages/client/components/ErrorBoundary.tsx:53]: Thanks! I always appreciate the itty bitty nitty tips

**Pull request merged**